src/rules/unbound-method.ts

```diff
@@ -68,7 +68,9 @@ export default createRule<Options, MessageIds>({
   name: __filename,
   meta: {
     messages: {
+      // eslint-disable-next-line eslint-plugin/no-unused-message-ids
```

Comment:
should we report a bug?

Reply:
Na these are being used by the base rule but we need to define them here to make TypeScript happy since it relates to what IDs we can use in the tests (they actually get overridden if the base rule exists).

I wouldn't expect the plugin to try and figure that out

Reply:
#1196

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/290